_test/shared/matlab_xunit_ISISextras/manage_legacy_sqw_class_rename.m

```diff
@@ -0,0 +1,23 @@
+function processed = manage_legacy_sqw_class_rename(data)
+% Perform pre-processing on data loaded from .mat files as part of the
+% TestCaseWithSave workflow.
+%
+% This is a temporary function to manage reengineering of the `sqw` object
+% during which the old class is renamed as `sqw_old`, and will load as
+% a struct rather than class instance.
+
+    if isfield(data, 'main_header') && isfield(data, 'header') && isfield(data, 'detpar')
+        processed = sqw_old(data);
```

Comment:
The function header isn't indented but the code is. I usually do not indent in top level functions - may as well save the horizontal whitespace.